### DISCUSSION TITLE
How to change jQuery global namespace

### DISCUSSION THREAD

**Discussion by ardimaunahan** [jquery/jquery#5337] [Q&A] [answered]
Hi. I'm planning to use the latest version (3.7.1) and in our old one (1.10.2), my team had to do search/replace the word "jquery" into something else so that we can have our own jQuery global namespace. Is there a much easier way now? Note that we are simply loading the file in the browser and not via module loaders like AMD. Thanks!

**Comment by mgol** [comment, marked as answer]
See https://api.jquery.com/jQuery.noConflict/
